bootstrap/Platform_Management/Setup_Custom_Cartridge_Library.groovy

```diff
@@ -0,0 +1,38 @@
+// Constants
+def libraryGitURL = "https://github.com/RobertNorthard/adop-dsl-custom-cartridge-properties.git"
```

Comment:
Please can the git url be update to use adop-dsl-custom-cartridge-properties library in the Accenture github org?